verifier/trustpolicy/trustpolicy.go

```diff
@@ -263,22 +267,29 @@ func LoadDocument() (*Document, error) {
 	policyDocument := &Document{}
 	err = json.NewDecoder(jsonFile).Decode(policyDocument)
 	if err != nil {
-		return nil, err
+		if errors.Is(err, io.EOF) {
+			return nil, errors.New("empty trustpolicy.json file. See a valid trustpolicy.json example at https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy")
+		}
```

Comment:
I have concern that `io.EOF` does not imply empty file. Are there any other situations that may return `io.EOF`? May be `early EOF`? or `early EOF` will just be a generic json decoder errror?

Reply:
Good point. I guess we could just remove this check? since it's one of the invalid file types anyway.